File: API/Entities/Scout.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Entities
{
    [Table("Scout")]
    public class Scout
    {
        [Key]
        public int MemberId { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public DateTime RechartedDate { get; set; }
        public bool Active { get; set; }
        public DateTime ActiveSinceDatetime { get; set; }
        public string PhotoUrl { get; set; }
        public string PublicId { get; set; }
        public int PatrolId { get; set; }
        public DateTime Created { get; set; } = DateTime.Now;

        public ICollection<Transaction> Transactions { get; set; }
        public ICollection<BuckTransaction> BuckTransactions { get; set; }
        public ICollection<Adult> Parents { get; set; }
        public ICollection<ScoutRank> ScoutRanks { get; set; }
    }
}

File: client/src/app/_models/scout.ts
import { ScoutRank } from './scoutRank';
import { Rank } from './rank';


import { Transaction } from "./transaction";
import { Adult } from './adult';

export interface Scout {

    memberId: number;
    lastName: string;
    firstName: string;
    age: number;
    rechartedDate: Date;
    active: boolean;
    activeSinceDatetime: Date;
    photoUrl: string;
    publicId?: string;
    rankName: string;
    patrolId: number;
    patrolName: string;
    created: Date;
    transactions: Transaction[];
}